### PR TITLE
add multicloud-operators-foundation repo

### PR DESCRIPTION
**Pull request by ldpliu**

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -252,6 +252,10 @@ approve:
   repos:
   - che-incubator/che-test-harness
   require_self_approval: false
+- commandHelpLink: https://go.k8s.io/bot-commands
```
Comment (droslean) [inline core-services/prow/02_config/_plugins.yaml:255]: This link is pointing to a different prow

File: core-services/prow/02_config/_plugins.yaml [hunk 2]
```diff
@@ -252,6 +252,10 @@ approve:
...
+  repos:
+  - open-cluster-management/multicloud-operators-foundation
```
Comment (qiujian16) [inline core-services/prow/02_config/_plugins.yaml:257]: i do not think you this, open-cluster-management as a whole has been included.

**Comment by qiujian16** [comment]
i think you also need to add verify-deps

**Pull request merged**